src/components/formreporttype.jsx

```diff
@@ -0,0 +1,129 @@
+        <h4>How Can We Help?</h4>
+        <Formik
+          initialValues={{
+            requestType: "",
+            subRequestType: ""
+          }}
+          validationSchema={Yup.object().shape({
+            requestType: Yup.string().required("Request Category is required"),
+            subRequestType: Yup.string().required("Sub Category is required")
+          })}
+          onSubmit={(values, { setSubmitting }) => {
+            setTimeout(() => {
+              alert(JSON.stringify(values, null, 2));
+              setSubmitting(false);
+            }, 400);
+          }}
+        >
+          {props => {
+            const { values, isSubmitting, errors, touched } = props;
+
+            return (
+              <Form onSubmit={props.handleSubmit}>
+                <label htmlFor="requestType">Request Category</label>
+                <br />
+                <Field
+                  component="select"
+                  id="requestType"
+                  name="requestType"
+                  onChange={e => {
+                    handleServiceRequestChange(e);
+                    props.setFieldValue(
+                      e.currentTarget.name,
+                      e.currentTarget.value
+                    );
+                    props.setFieldValue("subRequestType", "");
+                  }}
+                >
+                  <option key="default" value="">
+                    --Please select a category--
+                  </option>
+                  {Categories.map(category => (
+                    <option key={category.id} value={category.id}>
+                      {category.name}
+                    </option>
+                  ))}
+                </Field>
+
+                <div className="input-feedback">
+                  {
+                    <ErrorMsg
+                      errormessage={errors.requestType}
+                      touched={touched.requestType}
+                    />
+                  }
+                </div>
+
+                {values["requestType"] !== "" ? (
+                  <div>
+                    <label name="subRequestType" htmlFor="subRequestType">
+                      Request Sub-Category
+                    </label>
+                    <br />
+                    <Field
+                      component="select"
+                      id="subRequestType"
+                      name="subRequestType"
+                    >
+                      <option key="default" value="">
+                        --Please Select a sub-category--
+                      </option>
+                      ;
+                      {subCategories.map(category => (
+                        <option key={category.id} value={category.id}>
+                          {category.name}
+                        </option>
+                      ))}
+                    </Field>
+                    <div className="input-feedback">
+                      {
```

Comment:
you shouldn't need these brackets i don't think.

Reply:
brackets are removed